data/operators/power/generator.json

```diff
@@ -6443,8 +6443,8 @@
     },
     {
       "displayName": "エコ・パワー株式会社",
-      "id": "d72269-74685d",
-      "locationSet": {"include": ["001"]},
+      "id": "d72269-11cc64",
+      "locationSet": {"include": ["jp"]},
       "tags": {
         "operator": "エコ・パワー株式会社",
         "operator:ja": "エコ・パワー株式会社",
```

Comment:
If possible you can add a English version for operator because it is full of katagana, for example `"operator:en":"Eco Power Co., Ltd.",` ?